audio-worklet/src/bitcrusher-demo.js

```diff
@@ -102,20 +125,23 @@ class BitcrusherDemo {
         });
 
     // GUI controls allow switching between audio worklet and script processor.
-    // The audio worklet node is used by default.
-    this.useAudioWorkletCheckBox_ = document.createElement('input');
-    this.useAudioWorkletCheckBox_.type = 'checkbox';
-    this.useAudioWorkletCheckBox_.checked = true;
-    const audioworkletLabel = document.createElement('label');
-    audioworkletLabel.textContent =
-        'Use AudioWorklet (if unchecked, use ScriptProcessor Bitcrusher).';
-
-    this.useAudioWorkletCheckBox_.addEventListener(
-        'change', this.switchBitcrusherNode.bind(this));
-
-    let container = document.getElementById(containerId);
-    container.appendChild(this.useAudioWorkletCheckBox_);
-    container.appendChild(audioworkletLabel);
+    // The script processor node is used by default, and switching is not
+    // enabled if the browser does not support audio worklet.
+    if (this.useWorkletAndScriptProcessor_) {
```

Comment:
So, if AudioWorklet isn't available, you don't even show the checkbox?  Is that the right UI? I was kind of expecting the checkbox to be there, but unclickable.  So the user knows something is missing.